pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.openeo</groupId>
     <artifactId>openeo-opensearch-client</artifactId>
-    <version>1.6.2_2.13-SNAPSHOT</version>
+    <version>1.7.0_2.13-SNAPSHOT</version>
```

Comment:
Dropping parentheses from  `FeatureBuilder#build` is a breaking change -> 2.0.0_2.13-SNAPSHOT.